src/main.js

```diff
@@ -40,13 +42,14 @@ const App = Vue.extend( {
 //Define route for vue app
 //ref : http://router.vuejs.org/en/
 const router = new VueRouter( {
-	mode: 'history',
```

Comment:
@vishaldodiya 
Please don't modify existing code. In this case, mode history is needed to sync WordPress permalink structure with VueRoute.

Thanks